templates/README.md

```diff
@@ -0,0 +1,53 @@
+# UDS Common GitLab CI Components
+
+This folder contains the GitLab CI Components for UDS Common, analogous to the GitHub callable workflows under the `./github/workflows` directory. The included components are as follows:
+
+1. Lint
+1. Commitlint
+1. Test
+1. Publish
+1. Scorecard
+
+The .gitlab-ci.yml at the root of this repo serves as an example and test of how to call these components. That being said, a general example of how to include these components in a GitLab `uds-package` is as follows:
+
+```yaml
+include:
+  # include the component located in the current project from the current SHA
+  - component: $CI_SERVER_FQDN/$CI_PROJECT_PATH/commitlint@$CI_COMMIT_SHA
```

Comment:
If this is example usage for uds-packages in gitlab then isn't `CI_COMMIT_SHA` incorrect? They should be referring to a specific tag right?

Reply:
yes